src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -307,8 +312,21 @@ export const handleCreatePresignedPostUrlForLogos: RequestHandler<
   )
 }
 
+// Validates that the ending date is greater than the starting date
+const validateDateRange = celebrate({
+  [Segments.QUERY]: Joi.object()
+    .keys({
+      startDate: Joi.date().format('YYYY-MM-DD').raw(),
+      endDate: Joi.date()
+        .format('YYYY-MM-DD')
+        .greater(Joi.ref('startDate'))
```

Comment:
should be `min` instead of `greater` or there will be regression T_T, see #1634

Reply:
oh ok HAHAHAH will do!

Reply:
ported over tests from #1634 and also added a slightly modified version for `/submissions/download`